NodeJS_Course/countries-app/.gitignore

```
@@ -0,0 +1 @@
+node_modules/**/*
```

Esto también funciona si solo se usa:

```suggestion
node_modules
```

(¿Por qué? ¿Cuál es la diferencia entre ambos?)

---

Pensaba que era necesario para asegurarse que no añada ningun paquete que estaba por dentro, veo que no, gracias por la corrección!.